packages/botonic-react/src/components/multichannel/multichannel-text.jsx

```diff
@@ -96,14 +103,19 @@ export const MultichannelText = props => {
     return generator
   }
 
-  const splitPostbackButtons = postbackButtons => {
+  const splitInWhatsappListButtons = postbackButtons => {
     const messages = []
-    for (let i = 0; i < postbackButtons.length; i += WHATSAPP_MAX_BUTTONS) {
-      messages.push(postbackButtons.slice(i, i + WHATSAPP_MAX_BUTTONS))
+    for (
+      let i = 0;
+      i < postbackButtons.length;
+      i += WHATSAPP_LIST_MAX_BUTTONS
+    ) {
+      messages.push(postbackButtons.slice(i, i + WHATSAPP_LIST_MAX_BUTTONS))
     }
     return messages
   }
 
+  // START WHATSAPP LOGIC
   if (isWhatsapp(requestContext)) {
```

Comment:
Remove comment? I think the condition is clear enoguh to know we will enter the WhatsApp logic.